pdf-generator/src/main/java/uk/gov/hmcts/reform/pdf/generator/appinsights/AppInsightsEventTracker.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.pdf.generator.appinsights;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+
+import java.text.DecimalFormat;
+
+import static java.util.Collections.singletonMap;
+import static uk.gov.hmcts.reform.pdf.generator.appinsights.AppInsightsEvent.PDF_GENERATOR_FILE_NAME;
+import static uk.gov.hmcts.reform.pdf.generator.appinsights.AppInsightsEvent.PDF_GENERATOR_FILE_SIZE;
+
+public class AppInsightsEventTracker {
+
+    private static final String FILE_NAME = "file.name";
+    private static final String FILE_SIZE = "file.size";
+
+    private TelemetryClient telemetry;
+
+    public AppInsightsEventTracker() {
+        this.telemetry  = new TelemetryClient();
+    }
+
+    public AppInsightsEventTracker(TelemetryClient telemetry) {
+        this.telemetry = telemetry;
+    }
+
+    public void trackFileSize(float fileSize) {
+        telemetry.trackEvent(PDF_GENERATOR_FILE_SIZE.toString(), singletonMap(FILE_SIZE, convertSize(fileSize)), null);
+    }
+
+    public void trackFileName(String fileName) {
```

Comment:
this method doesn't seem to be called, but I think recording it is likely too risky as I know cmc was putting peoples names in the files...

Reply:
OK... I will remove this ... I wasn't sure either.